lib/quantcast/index.js

```diff
@@ -156,38 +194,37 @@ Quantcast.prototype.completedOrder = function(track){
 };
 
 /**
- * Generate quantcast labels.
- *
- * Example:
- *
- *    options.advertise = false;
- *    labels('event', 'my event');
- *    // => "event.my event"
+ * Helper method to generate quantcast labels.
  *
- *    options.advertise = true;
- *    labels('event', 'my event');
- *    // => "_fp.event.my event"
- *
- * @param {String} type
- * @param {String} ...
+ * @param {String} prefix
+ * @param {Array} pageMeta
+ * @param {Array} ...
```

Comment:
Ahh, that's a leftover from the previous iteration. It should be `[customfields]`.